cypress/integration/pages/frontPage/testsForCanonicalOnly.js

```diff
@@ -11,6 +13,23 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({ service }) => {
   if (Cypress.env('APP_ENV') === 'local') {
     runCanonicalAdsTests({ service });
   }
+
+  describe(`US Election Banner`, () => {
+    it('should have US Election banner if toggle is enabled', () => {
+      cy.getToggles(config[service].name);
```

Comment:
is it possible to use the `service` value passed it on line 12 ? instead of `config[service].name`

Reply:
Service is not always the correct service name - the value comes from https://github.com/bbc/simorgh/blob/latest/cypress/support/config/settings.js#L2, and if you look at any of the variant services, this does not match the service name e.g. `ukChinaTrad` instead of `ukchina` (it's very 😕 I know)